app/subscriber/src/features/my-reports/edit/ReportEditActions.tsx

```diff
@@ -155,7 +155,10 @@ export const ReportEditActions = ({ disabled, updateForm }: IReportEditActionsPr
         {!disabled ||
         instance?.status === ReportStatusName.Pending ||
         active?.startsWith(ReportMainMenuOption.Settings) ? (
-          <Button onClick={() => submitForm()} disabled={isSubmitting}>
+          <Button
```

Comment:
Don't allow saving when a report is currently being processed by the Reporting Service.